<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="whatsThis">
         <string extracomment="Color boost"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string extracomment="Color Boost"/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="sliderPosition">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSetDefaultPosition"/>
   </widget>
   <widget class="QMenu" name="menuEffect_Type">
    <property name="title">
     <string>Effect Type</string>
    </property>
    <addaction name="actionMandelProt_Fractal"/>
    <addaction name="actionZ_Z_Z_Z_C"/>
    <addaction name="actionJulian_SetPreview"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuEffect_Type"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetDefaultPosition">
   <property name="text">
    <string>SetDefaultPosition</string>
   </property>
  </action>
  <action name="actionMandelProt_Fractal">
   <property name="text">
    <string>MandelProt Fractal</string>
   </property>
  </action>
  <action name="actionZ_Z_Z_Z_C">
   <property name="text">
    <string>Z = Z*Z*Z + C</string>
   </property>
  </action>
  <action name="actionJulian_SetPreview">
   <property name="text">
    <string>Julian SetPreview</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
